Governance/tech/keywords.md

```diff
@@ -1,16 +1,17 @@
 # Choosing keywords
 
-There are few categories you should try to cover when assigning keywords:
+There are a few categories you should try to cover when assigning keywords:
 
 * dataset acronym: if your data is strictly related to another dataset, or your code is applied to a specific dataset
 * model acronym and version: as for datasets if you generated the data using a model
 * project acronym: if your dataset and or code relates to a specific project 
 * programming language:  you should add this to your code records and be as specific as possible, for example use python3, rather than just python
 * data type: observation, model output, etc.
-* realm or discipline: like ocean, land and/or physical ocanography, climate science etc. For the disciplines you can use the [Fields of Research codes](http://climate-cms.wikis.unsw.edu.au/Field_of_Research_codes) from the Bureau of statistics
+* realm or discipline: like ocean, land and/or physical oceanography, climate science etc. For the disciplines you can use the [Fields of Research codes](http://climate-cms.wikis.unsw.edu.au/Field_of_Research_codes) from the Bureau of Statistics
```

Comment:
Australian Bureau of Statistics maybe?

Reply:
Agreed - should add Australian

Reply:
yes good point, I'm also noting here that probably I should add that wiki page here, it's a summary of the climate related FOR code from the new version. I'm just commenting on this, I didn't have time tor review this today, but I will on Monday